.env.example

```
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_CLOUD_API_URL=http://localhost:3111/api
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key should go before the NEXT_PUBLIC_CLOUD_API_URL key

(UnorderedKey)